lib/block_account.go

```diff
@@ -21,6 +21,8 @@ import (
 
 const BlockAccountPrefixAddress string = "ba-address-"
 const BlockAccountPrefixCreated string = "ba-created-"
+const BlockAccountCheckpointPrefix string = "bac-ac-"
```

Comment:
why do we need those prefix?
To avoid hash conflict?

Reply:
@kfangw Yes, right.